magicarp/pipeline/harmless_base_pipeline.py

```diff
@@ -0,0 +1,48 @@
+import json
+import gzip
+from magicarp.pipeline import Pipeline, register_datapipeline
+
+@register_datapipeline
+class HarmlessBasePipeline(Pipeline):
+    def __init__(self, path):
```

Comment:
Add `super().__init__()` at start of constructor to make sure this calls `Pipeline.__init__()` and inherits its methods/attributes.